### PR TITLE
gatsby/sanity connection and font prefetch feature/#23

### PR DESCRIPTION
**Pull request by mikkelcodes**
Font prefetching og sanity config i gatsby. Det er nå mulig å query data fra sanity-prosjektet (localhost:8000/__graphql)

### CODE REVIEW

File: packages/website/gatsby-config.js [hunk 1]
```diff
@@ -1,3 +1,30 @@
+require('dotenv').config({
+  path: `.env.${process.env.NODE_ENV}`,
```
Comment (TheKnarf) [inline packages/website/gatsby-config.js:2]: What does these two lines do?

File: packages/website/package.json [hunk 2]
```diff
@@ -7,11 +7,15 @@
...
+    "add": "^2.0.6",
     "gatsby": "^2.25.2",
     "gatsby-plugin-postcss": "^3.0.4",
+    "gatsby-plugin-prefetch-google-fonts": "^1.4.3",
+    "gatsby-source-sanity": "^6.0.4",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "tailwindcss": "^1.9.6"
+    "tailwindcss": "^1.9.6",
+    "yarn": "^1.22.10"
```
Comment (TheKnarf) [inline packages/website/package.json:18]: How did yarn become a dependecy here?

File: packages/website/package.json [hunk 3]
```diff
@@ -7,11 +7,15 @@
     "build:storybook": "build-storybook --no-dll --quiet"
   },
   "dependencies": {
+    "add": "^2.0.6",
```
Comment (TheKnarf) [inline packages/website/package.json:10]: Where do we use this?

**Review by TheKnarf — CHANGES_REQUESTED** [review]
Perhaps we could split this into two commits? One for the `font` issue and one for the `sanity plugin`?

File: packages/website/package.json (hunk 2 above)
Comment (TheKnarf) [reply packages/website/package.json:18]: I think perhaps you accidentally ran `yarn add yarn add` and thereby added `yarn` and `add` as a dependency.
You should be able to fix it by removing those two dependencies, re-run `yarn` and then add and commit it with amend :)

File: packages/website/gatsby-config.js (hunk 1 above)
Comment (mikkelcodes) [reply packages/website/gatsby-config.js:2]: It allows gatsby-config to access the following file packages/website/.env.development which contains our environment variables (Sanity access token).

.env.development, which should be ignored by .gitignore, should contain one line:
SANITY_TOKEN="token goes here"

Sanity picks up this token on line 11 in gatsby-config.js and this allows for draft previews. I can demo it later when I have some content to preview. I also forgot to add "overlayDrafts: true" to the config, so I will include that in the upcoming ----amend 👍

File: packages/website/package.json (hunk 2 above)
Comment (mikkelcodes) [reply packages/website/package.json:18]: Yup, that's probably what happened here. Got it

**Comment by mikkelcodes** [comment]
> Perhaps we could split this into two commits? One for the `font` issue and one for the `sanity plugin`?
Didn't see your comment here before I pushed the amended commit. If you agree with all changes, could we let it slide this time and I'll keep all future PRs specific to each single issue?

**Review by TheKnarf — APPROVED** [review]

**Pull request merged**